app/components/provider_interface/application_card_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="app-application-card">
+
+  <div class="app-application-card__primary">
+    <h3 id="applicant-name-<%= index %>" class="govuk-heading-m govuk-!-margin-bottom-3">
+      <%= link_to candidate_name, provider_interface_application_choice_path(application_choice), class: 'govuk-link' %>
+    </h3>
+    <p id="course-provider-name-<%= index %>" class="govuk-body govuk-!-font-size-16"><%= course_provider_name %></p>
+    <p id="course-name-and-code-<%= index %>" class="govuk-body govuk-!-margin-bottom-0"><%= course_name_and_code %></p>
+    <p id="accredited-body-<%= index %>" class="govuk-body govuk-caption-m govuk-!-font-size-16">Accredited body: <%= accrediting_provider.name if accrediting_provider %></p>
```

Comment:
The markup for this component could be improved by not using `<p>` for the nested pieces of data. There are a few downsides currently:

- Without CSS, the hierarchy of the data could be unclear
- Screen reader users could have trouble figuring out what the values in the different paragraphs are meant to map to; apart from the `Accredited body:` which would be clear

Would you consider trying out 	[\<dl\> for this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)? Could look like:

```html
<h3>Kelly Jane Spears</h3>

<dl>
  <dt class="govuk-visually-hidden">Provider:</dt>
  <dd>Sommerset SCITT Consortium</dd>

  <dt class="govuk-visually-hidden">Course:</dt>
  <dd>Primary (X100)</dd>

  <dt>Accredited body:</dt>
  <dd>Sommerset SCITT Consortium</dd>

  <dt>Updated on</dt>
  <dd>10 Oct 2019</dd>
</dl>
```

Might require to re-jigger the CSS to lay it out as per the design.

@adamsilver WDYT of this suggestion?

Reply:
👍 - copied from the design so unless @adamsilver has a reason for using `<p>` tags I'll refactor.